greenheart/simulation/greenheart_simulation.py

```diff
@@ -818,7 +887,30 @@ def simple_solver(initial_guess=0.0):
         return hopp_results, electrolyzer_physics_results, remaining_power_profile
     elif config.output_level == 7:
         return lcoe, lcoh, steel_finance, ammonia_finance
-
+    elif config.output_level == 8:
```

Comment:
What do you think about getting rid of `output_level` completely and just returning the data class? I know that it might mess a few folks up right now but lots of other breaking changes are happening on this branch and I think using data classes will help improve the greenheart simulation generalization

Reply:
I would like to avoid running post-processing and passing a lot of extra stuff around during optimization runs. So, for now anyway, I would like to keep the output levels. We should probably make output level 8 the de facto in our example yamls though to encourage people to use that option